examples/python/deep_sdf/requirements.txt

```diff
@@ -1,4 +1,4 @@
-mesh_to_sdf===0.0.14
+-e git+https://github.com/marian42/mesh_to_sdf.git#egg=mesh_to_sdf
```

Comment:
wth is this magic? "egg"? https://youtu.be/hl7-ZmJEB1E?t=24

Is it possible to add a comment here explaining wtf is going on and why we don't just depend on `mesh_to_sdf` like a normal dependency?

Reply:
I watched the video you linked and I still don't know what egg is.

I'm just following Pip's orders. But apparently that's what bundled python code is called https://stackoverflow.com/questions/2051192/what-is-a-python-egg
Turns out.. pythons.. lay eggs..

I also found out how to add comments to requirements.txt so I'll link to the ticket etc.